### DISCUSSION TITLE
Pinned Disk IO v0.34.0

### DISCUSSION THREAD

**Discussion by NeoKactus** [dokku/dokku#6767] [Q&A] [answered]
I tried upgrading from v0.33.9 to v0.34.0 and noticed that my deployment time tripled due to disk IO. 
The disk IO is being pinned by `chown -R herokuishuser:herokuishuser /app`. I am assuming it's trying to chown node_modules on our HDDs. 

I still need to play around with the issue, but for now we downgraded to v0.33.9. Is there something I am missing from the migration guide or should I file a bug report with additional info?

Side question. Is it practical/feasible to setup a ramdisk for the Dokku build process?

**Comment by josegonzalez** [comment, marked as answer]
I think this was caused by a failure to release the herokuish repo, which is currently blocked by a bug. I'm [investigating a fix](https://github.com/gliderlabs/herokuish/pull/1160) now and will report back. Thanks for raising this!

**Reply by josegonzalez** [reply]
This change should also help a bit: https://github.com/gliderlabs/herokuish/pull/1173/files

That said, I think I might also need to place that behind the same check, so we'll see how it impacts things.

**Comment by josegonzalez** [comment]
@NeoKactus this should be fixed now if you upgrade to at least Herokuish 0.8.0 and Dokku 0.34.1.

**Reply by NeoKactus** [reply]
Ty that has fixed it!
